templates/ai/mlops-workshop.html

```diff
@@ -0,0 +1,329 @@
+{% extends "ai/base_ai.html" %}
+
+{% block title %}MLOps workshop{% endblock %}
+
+{% block meta_description %}
+  Accelerate your AI/ML projects with the MLOps workshop hosted by Canonical open source experts.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1g06uHWdy2ohYq-tKCEsdlD2EohZQ8OHSWO3i8U5vUDc/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% from "macros/_macros-hero.jinja" import hero %}
+{% from "macros/_macros-text-list.jinja" import text_list %}
+{% from "macros/_macros-highlights.jinja" import highlights %}
+{% from "macros/_macros-text.jinja" import text %}
+{% from "macros/_macros-quote.jinja" import quote_wrapper %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+{% from "macros/_macros-cta.jinja" import cta %}
+
+{% block content %}
+
+  {%- call(slot) hero(
+    title="Accelerate your AI/ML projects with an MLOps workshop",
+    layout='50/50',
+    is_split_on_medium="true"
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Build your tailored MLOps architecture in just 5 days. Our workshop will help you design AI infrastructure for any use case &ndash; from the data center to the edge.  Accelerate and scale your AI/ML initiatives, optimize resource usage and elevate your in-house expertise.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a class="p-button--positive js-invoke-modal"
+         href="/ai/mlops-workshop#get-in-touch">Contact the team</a>
+      <a href="https://pages.ubuntu.com/rs/066-EOV-335/images/Datasheet%20for%20MLOps%20workshop.pdf?version=0">Download the datasheet&nbsp;&rsaquo;</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="u-embedded-media">
+        <iframe title="Open Source AI innovation with Canonical"
+                class="u-embedded-media__element"
+                src="https://www.youtube.com/embed/yerW2pOczvA"
+                allowfullscreen=""
+                frameborder="0"></iframe>
+      </div>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <h2 class="p-section--shallow">Why join an MLOps workshop?</h2>
+    </div>
+    <div class="row--25-75 is-split-on-medium">
```

Comment:
Sorry, I think mine and Mattea's comments [here](https://github.com/canonical/ubuntu.com/pull/14474#discussion_r1857054873) were missed. We should use offsets here instead of an empty div and define medium cols which won't work with the `25-75` grid short hard. Suggested edits below: